.github/quarkus-github-bot.yml

```
@@ -531,8 +531,8 @@ triage:
     - labels: [area/optaplanner]
       title: "optaplanner"
       notify: [ge0ffrey, rsynek, Christopher-Chianelli]
-    - labels: [kind/extension-proposal]
-      title: quarkiverse
+    - labels: [kind/extension-proposal,area/quarkiverse]
+      title: "[Extension Proposal]"
```

This is interpreted as a regex so better not make it a character class so either that fix, or you have to escape the regex properly (I don't think it's worth the hassle):
```suggestion
      title: "Extension Proposal"
```